Tensile/Common.py

```diff
@@ -2335,7 +2342,7 @@ def assignGlobalParameters( config ):
 
   if "TENSILE_ROCM_ASSEMBLER_PATH" in os.environ:
     globalParameters["AssemblerPath"] = os.environ.get("TENSILE_ROCM_ASSEMBLER_PATH")
-  elif globalParameters["AssemblerPath"] is None and supportedLinuxCompiler(globalParameters["CxxCompiler"]):
+  elif globalParameters["AssemblerPath"] is None and supportedCompiler(globalParameters["CxxCompiler"]):
     if os.name == "nt":
```

Comment:
Not sure we even care if "and supportedCompiler" at this conditional, but fine

Reply:
I believe some of this logic i.e. and supportedCompiler is related to supporting opencl. We can create another ticket to investigate and remove.